lib/augmentConfig.js

```diff
@@ -98,25 +122,30 @@ function augmentConfigFull(
 		});
 }
 
-// Make all paths in the config absolute:
-// - ignoreFiles
-// - plugins
-// - processors
-// (extends handled elsewhere)
+/**
+ * Make all paths in the config absolute:
+ * - ignoreFiles
+ * - plugins
+ * - processors
+ * (extends handled elsewhere)
+ * @param {StylelintConfig} config
+ * @param {string} configDir
+ * @returns {StylelintConfig}
+ */
 function absolutizePaths(
 	config /*: stylelint$config*/,
 	configDir /*: string*/,
 ) /*: stylelint$config*/ {
 	if (config.ignoreFiles) {
-		config.ignoreFiles = [].concat(config.ignoreFiles).map((glob) => {
+		config.ignoreFiles = /** @type {string[]} */ ([]).concat(config.ignoreFiles).map((glob) => {
```

Comment:
What is the difference between `Array<string>` and `string[]`?

Reply:
I think there is no difference, as I understand array literal `[]` is build-in, `Array` is generic but they follows same logic under the hood.